otx/cli/tools/train.py

```diff
@@ -128,6 +128,18 @@ def parse_args():
         type=float,
         help="Expected ratio of total time to run HPO to time taken for full fine-tuning.",
     )
+    parser.add_argument(
+        "--gpus",
+        type=str,
+        help="Comma-separated indcies of GPU. \
+              If there are more than one available GPU, then model is trained with multi GPUs.",
+    )
+    parser.add_argument(
+        "--multi-gpu-port",
```

Comment:
Does the user need to set this?

Reply:
yes, if user already set those port. and if user want to run multi multi-GPU training parallelly, then user should set different port to each training.

Reply:
Any guide on this? If the port is the same, then we have two processes on the same gpu, and if they are given differently, we can run them on different gpu?

And what does default mean? (I don't really know much about this part. I hope you understand.)

Reply:
Maybe some guides need to be added. I'll add some guides.
multi processes in a same multi-GPU training need to use same port.
multi processes, each belonging to different multi-GPU training, should use different port.
default means literally default value of argument. So, if user doesn't set this argument but set `--gups` to run multi-GPU training, then 25000 port number will be used.